src/test/java/qna/domain/AnswerRepositoryTest.java

```diff
@@ -0,0 +1,34 @@
+package qna.domain;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+
+import java.util.Optional;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@DataJpaTest
+public class AnswerRepositoryTest {
+    @Autowired
+    private AnswerRepository answerRepository;
+
+    public static final Answer A1 = new Answer(1L,UserTest.JAVAJIGI, QuestionTest.Q1, "Answers Contents1");
+    public static final Answer A2 = new Answer(2L, UserTest.SANJIGI, QuestionTest.Q1, "Answers Contents2");
+
+    @Test
+    public void save(){
```

Comment:
```suggestion
    public void save() {
```

https://google.github.io/styleguide/javaguide.html#s4.6.2-horizontal-whitespace

다른 코드도 같이 수정해보아요

Reply:
훔훔.. 코드 스타일맞추고, 포맷을 돌리는데도 하나씩 걸리네용 ㅠ